commands/version/command.js

```diff
@@ -263,12 +263,6 @@ exports.builder = (yargs, composed) => {
         delete argv.githubRelease;
         log.warn("deprecated", "--github-release has been replaced by --create-release=github");
       }
-      /* eslint-enable no-param-reassign */
-
-      if (argv["--"]) {
-        log.warn("EDOUBLEDASH", "Arguments after -- are no longer passed to subprocess executions.");
-        log.warn("EDOUBLEDASH", "This will cause an error in a future major version.");
-      }
```

Comment:
Not sure if this causes any other problems?

Reply:
This command does not support `--` args passthrough because it is not clear which subprocesses those arguments are being passed to. Just by running `lerna version -- --force`, it is not clear that the `--force` is only relevant for an inner `npm install` command. This is why more specific options (like --npmClientArgs) are preferred for specifying subprocess arguments.